packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -52,15 +40,9 @@ const propTypes = {
    * An ISO 8601 string representation of the initial default date to show in the date input. This prop name is derived from react-datepicker but is analogous to defaultValue for a form input field.
    */
   selectedDate: PropTypes.string,
-  /**
-   * An ISO 8601 string representation of the start date for a date range.
-   */
-  startDate: PropTypes.string,
 };
 
 const defaultProps = {
```

Comment:
Yeah, there will be merge conflict when these are ready to be merged. I'll make sure that any reference to date range in that PR doesn't get merged down.